docs/develop/rest-api/proposed/symbols-api.md

```
@@ -0,0 +1,369 @@
+---
+title: Symbols API
+---
+
+# Working with the Symbols API
+
+#### (Proposed)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)